### PR TITLE
Add Meeting Notes for Jan 10 2020

### LINKED ISSUE #563: Node.js Foundation Community Committee Meeting 2020-01-09
## Time

**UTC Thu 09-Jan-2020 17:00 (05:00 PM)**:

| Timezone      | Date/Time             |
|---------------|-----------------------|
| US / Pacific  | Thu 09-Jan-2020 09:00 (09:00 AM) |
| US / Mountain | Thu 09-Jan-2020 10:00 (10:00 AM) |
| US / Central  | Thu 09-Jan-2020 11:00 (11:00 AM) |
| US / Eastern  | Thu 09-Jan-2020 12:00 (12:00 PM) |
| London        | Thu 09-Jan-2020 17:00 (05:00 PM) |
| Amsterdam     | Thu 09-Jan-2020 18:00 (06:00 PM) |
| Moscow        | Thu 09-Jan-2020 20:00 (08:00 PM) |
| Chennai       | Thu 09-Jan-2020 22:30 (10:30 PM) |
| Hangzhou      | Fri 10-Jan-2020 01:00 (01:00 AM) |
| Tokyo         | Fri 10-Jan-2020 02:00 (02:00 AM) |
| Sydney        | Fri 10-Jan-2020 04:00 (04:00 AM) |

Or in your local time:
* http://www.timeanddate.com/worldclock/fixedtime.html?msg=Node.js+Foundation+Community%20Committee+Meeting+2020-01-09&iso=20200109T17
* or http://www.wolframalpha.com/input/?i=05PM+UTC%2C+Jan+09%2C+2020+in+local+time

## Links

* Minutes Google Doc: <https://docs.google.com/document/d/1tKHQ9kOvBBvzK6TmrAm0MPqzvxnVC7a-U1kr-fsNdSY/edit>

## Agenda

Extracted from **cc-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.

### nodejs/admin

* Node.js Social Media Accounts [#454](https://github.com/nodejs/admin/issues/454)

### nodejs/community-committee

* Remove individual membership director role and move williamkapke to emeritus [#560](https://github.com/nodejs/community-committee/pull/560)
* Add licenses to CommComm repositories [#559](https://github.com/nodejs/community-committee/issues/559)
* Starting an Examples Initiative [#550](https://github.com/nodejs/community-committee/issues/550)
* Community Committee and OKRs [#547](https://github.com/nodejs/community-committee/issues/547)
* Initiative #507 — Preparing questions for annual survey [#538](https://github.com/nodejs/community-committee/issues/538)
* Node.js Social Media Community Collaboration [#444](https://github.com/nodejs/community-committee/issues/444)

### nodejs/nodejs-collection
* Update readme with new process [#37](https://github.com/nodejs/nodejs-collection/pull/37)

## Invited

* CommComm Members: @nodejs/community-committee

### Observers/Guests
Feel free to follow along on the YouTube live stream, or attend meeting as a guest 
by calling in to Zoom, using the links below. If you will be attending as a guest, 
please comment on this issue to let us know you'll be joining.

## Notes

The agenda comes from issues labelled with `cc-agenda` across **all of the repositories in the nodejs org**. Please label any additional issues that should be on the agenda before the meeting starts.

## Joining the meeting

* Link for participants: https://zoom.us/j/523381536
* For those who just want to watch: https://www.youtube.com/channel/UCQPYJluYC_sn_Qz_XE-YbTQ

---

**Invitees**

Please use the following emoji reactions in this post to indicate your
availability.

* :+1: - Attending
* :-1: - Not attending
* :confused: - Not sure yet

### PR DESCRIPTION
**Pull request by SauloNunes**
This closes #563

### CODE REVIEW

File: meetings/2020-01-09.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+* Waleed - In process with updating the Node.js Collection submission process and making it more open and easy to submit.
+
+## CPC and Board Meeting Updates
+
+*Extracted from **cc-agenda** labeled issues and pull requests from the **nodejs org** prior to the meeting.
+
+* Board - not too much to update here
+* CPC - Discussions around code of conduct and moderation.  New team/working group will
+   likely spin up. In terms of CoC agreement that will inline Contributor Covenant.
```
Comment (mhdawson) [inline meetings/2020-01-09.md]:
``` suggestion
  likely spin up. In terms of CoC agreement that will inline Contributor Covenant.
```

File: meetings/2020-01-09.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+* Weigh in on email related to naming of next iteration of event (Planned for June).  
+* Likely first collab summit of year will be in Austin with the conference, some discussion but still
+   at the “ideas” stage for where/when to do 2nd European collab summit which might be
```
Comment (mhdawson) [inline meetings/2020-01-09.md]:
```suggestion
  at the “ideas” stage for where/when to do 2nd European collab summit which might be
```

File: meetings/2020-01-09.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+   Node.js specific.
```
Comment (mhdawson) [inline meetings/2020-01-09.md]:
```suggestion
  Node.js specific.
```

**Comment by SauloNunes** [comment]
Thanks for Reviewing @mhdawson

**Pull request merged**